README.md

```diff
@@ -104,6 +104,54 @@ kubectl --kubeconfig=kubeconfig get machines
 kubectl --kubeconfig=kubeconfig get machines -o yaml
 ```
 
+### Debugging your cluster
+
+There are various services running that help to communicate with openstack and stand up your cluster. Most of these services have fairly robust logs which should be your first resort for debugging the application. 
+
+Primarily, the log level of clusterctl can be increased to show more verbose logging. To do this, simply append `-v#` where the `#` can be any number [1 - 10]. 
+
+If your error is not here, there is a strong chance that it will be in the openstack-machine-controller pod. To check the logs of this pod, follow these steps:
+
+```bash
+minikube update-context
+kubectl get pods --all-namespaces
+```
+
+At this point, you will have a list of all the pods in the minikube cluster. Now, you need to get the logs of the openstack-machine-controller pod. Look for the pod named `clusterapi-controllers-...` in the list of pods returned from clusterctl. Now, use this command to get its logs:
```

Comment:
sometimes, the pod might be dead state, we may state use 
docker ps -a

Reply:
can docker see the pods in minikube?

Reply:
@jichenjc can you elaborate a little more what you want me to change